Server/models/user.model.js

```diff
@@ -14,9 +14,19 @@ module.exports = function (sequelize, DataTypes) {
 			 validate: {len: [2,50], isEmail: true},
 			 unique : true},
 			
-		isAdmin: { type: DataTypes.BOOLEAN, allowNull: false, defaultValue: false }
-		},	{
-		freezeTableName: true,
+		isAdmin: { type: DataTypes.BOOLEAN, allowNull: false, defaultValue: false },
+		userPasswd: {type: DataTypes.STRING, allowNull: false, len: [2,50]} 
+		},	
+		
+		{instanceMethods: {
+			  toJSON: function () {
+				var values = Object.assign({},
+					 this.get());		  
+				delete values.userPasswd;
+				return values;
+			  }}},
+			
+		{freezeTableName: true,
```

Comment:
are you sure, this should be placed in argument separate from the one with instanceMethods field?